### PR TITLE
Pass an io.Writer to NewErrorHook

### PR DESCRIPTION
**Pull request by caglar10ur**

### CODE REVIEW

**Comment by fdawg4l** [comment]
LGTM

File: cmd/imagec/hook.go [hunk 1]
```diff
@@ -16,25 +16,25 @@ package main
...
 import (
 	"fmt"
-	"os"
+	"io"
 
 	"github.com/Sirupsen/logrus"
 )
 
 // ErrorHook is a simple logrus hook to duplicate Fatalf leveled error messages to stderr
 type ErrorHook struct {
```
Comment (fdawg4l) [inline cmd/imagec/hook.go]: LGTM but do you think we should make this a pkg?  This looks pretty useful.

**Comment by dougm** [comment]
lgtm

**Comment by caglar10ur** [comment]
♻️

File: cmd/imagec/hook.go (hunk 1 above)
Comment (caglar10ur) [inline cmd/imagec/hook.go]: We can promote it to a pkg if someone else decides to use it, for now it's only used by imagec

**Pull request merged**